SPECS/ruby/ruby.spec

```diff
@@ -124,6 +122,8 @@ Provides:       rubygem-base64 = %{base64_version}-%{release}
 Provides:       rubygem(base64) = %{base64_version}-%{release}
 Provides:       rubygem-benchmark = %{benchmark_version}-%{release}
 Provides:       rubygem(benchmark) = %{benchmark_version}-%{release}
+Provides:       rubygen-bigdecimal = %{bigdecimal_version}-%{release}
+Provides:       rubygen(bigdecimal) = %{bigdecimal_version}-%{release}
 Provides:       rubygem-cgi = %{cgi_version}-%{release}
```

Comment:
Please add 'provides' for rubygem-bundler

Reply:
Fixed